python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -352,6 +352,7 @@ def __init__(
         settings: Optional[Mapping[str, Any]] = None,
         secrets_loader: Optional["SecretsLoader"] = None,
         ref: Optional[InstanceRef] = None,
+        **_kwargs: Any, # we accept kwargs for forward-compat of custom instances
```

Comment:
Would be useful to logs these "Ignoring fields {_kwargs.keys()}" if possible.

Reply:
Yea good idea.